<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabContainer</class>
 <widget class="QWidget" name="TabContainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1448</width>
    <height>54</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>54</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>54</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <widget class="QPushButton" name="preBtn">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>36</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addBtn">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="nextBtn">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>36</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
